sdk/core/src/config.rs

```diff
@@ -81,10 +80,7 @@ impl DocDbConfig {
                 "{}/?ssl=true&tlsCAFile=rds-combined-ca-bundle.pem&retryWrites=false",
                 connection_uri
             ),
-            false => format!(
-                "{}/?ssl=false&retryWrites=false",
-                connection_uri
-            )
+            false => format!("{}/?ssl=false&retryWrites=false", connection_uri),
```

Comment:
Is this using a linter rule, if not can it be?

Reply:
See previous comment about linter and splitting large commits.